pytransform3d/test/test_transform_manager.py

```diff
@@ -4,11 +4,14 @@
 import tempfile
 import numpy as np
 from pytransform3d.rotations import (
```

Comment:
Would it make sense to import the namespace as `tf3d_rot` and call the functions in the unittest?